lib/chef/cookbook/cookbook_version_loader.rb

```diff
@@ -231,12 +231,12 @@ def load_all_files
 
             Dir.glob(File.join(fs_entry, "**/*"), File::FNM_DOTMATCH).each do |file|
               next if File.directory?(file)
-              file = Chef::Util::PathHelper.cleanpath(file)
+              file =  ::Pathname.new(file).cleanpath
```

Comment:
indentation.  i don't think you need the :: in front of Pathname either, i was just being pedantic to be clear i was referring to the stdlib on slack...